{"code":"ERR_HTTP_HEADERS_SENT","level":"error","message":"Unhandled error: Cannot set headers after they are sent to the client","stack":"Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (node:_http_outgoing:699:11)\n    at ServerResponse.header (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:794:10)\n    at ServerResponse.location (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:925:15)\n    at ServerResponse.redirect (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:963:18)\n    at file:///C:/Users/Punit/Desktop/ApiExplorer-Backend/src/routes/auth.routes.js:125:11\n    at Layer.handle [as handle_request] (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at complete (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\middleware\\authenticate.js:280:13)\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\middleware\\authenticate.js:287:15\n    at pass (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\authenticator.js:446:14)","timestamp":"2025-05-10T18:20:40.441Z"}
{"level":"error","message":"Unhandled error: Failed to obtain access token","stack":"Error: Failed to obtain access token\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport-oauth2\\lib\\strategy.js:178:49\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\oauth\\lib\\oauth2.js:214:7\n    at passBackControl (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\oauth\\lib\\oauth2.js:134:9)\n    at IncomingMessage.<anonymous> (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\oauth\\lib\\oauth2.js:157:7)\n    at IncomingMessage.emit (node:events:530:35)\n    at endReadableNT (node:internal/streams/readable:1698:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-05-10T18:20:49.589Z"}
{"level":"error","message":"Unhandled error: Failed to obtain access token","stack":"Error: Failed to obtain access token\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport-oauth2\\lib\\strategy.js:178:49\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\oauth\\lib\\oauth2.js:214:7\n    at passBackControl (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\oauth\\lib\\oauth2.js:134:9)\n    at IncomingMessage.<anonymous> (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\oauth\\lib\\oauth2.js:157:7)\n    at IncomingMessage.emit (node:events:530:35)\n    at endReadableNT (node:internal/streams/readable:1698:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-05-10T18:20:56.566Z"}
{"code":"ERR_HTTP_HEADERS_SENT","level":"error","message":"Unhandled error: Cannot set headers after they are sent to the client","stack":"Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (node:_http_outgoing:699:11)\n    at ServerResponse.header (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:794:10)\n    at ServerResponse.location (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:925:15)\n    at ServerResponse.redirect (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:963:18)\n    at file:///C:/Users/Punit/Desktop/ApiExplorer-Backend/src/routes/auth.routes.js:125:11\n    at Layer.handle [as handle_request] (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at complete (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\middleware\\authenticate.js:280:13)\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\middleware\\authenticate.js:287:15\n    at pass (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\authenticator.js:446:14)","timestamp":"2025-05-10T18:21:09.268Z"}
{"code":"ERR_HTTP_HEADERS_SENT","level":"error","message":"Unhandled error: Cannot set headers after they are sent to the client","stack":"Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (node:_http_outgoing:699:11)\n    at ServerResponse.header (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:794:10)\n    at ServerResponse.location (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:925:15)\n    at ServerResponse.redirect (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:963:18)\n    at file:///C:/Users/Punit/Desktop/ApiExplorer-Backend/src/routes/auth.routes.js:125:11\n    at Layer.handle [as handle_request] (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at complete (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\middleware\\authenticate.js:280:13)\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\middleware\\authenticate.js:287:15\n    at pass (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\authenticator.js:446:14)","timestamp":"2025-05-10T18:21:31.369Z"}
{"code":"ERR_HTTP_HEADERS_SENT","level":"error","message":"Unhandled error: Cannot set headers after they are sent to the client","stack":"Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (node:_http_outgoing:699:11)\n    at ServerResponse.header (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:794:10)\n    at ServerResponse.location (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:925:15)\n    at ServerResponse.redirect (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:963:18)\n    at file:///C:/Users/Punit/Desktop/ApiExplorer-Backend/src/routes/auth.routes.js:125:11\n    at Layer.handle [as handle_request] (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at complete (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\middleware\\authenticate.js:280:13)\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\middleware\\authenticate.js:287:15\n    at pass (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\authenticator.js:446:14)","timestamp":"2025-05-10T18:22:52.798Z"}
{"level":"error","message":"Unhandled error: Failed to obtain access token","stack":"Error: Failed to obtain access token\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport-oauth2\\lib\\strategy.js:178:49\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\oauth\\lib\\oauth2.js:214:7\n    at passBackControl (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\oauth\\lib\\oauth2.js:134:9)\n    at IncomingMessage.<anonymous> (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\oauth\\lib\\oauth2.js:157:7)\n    at IncomingMessage.emit (node:events:530:35)\n    at endReadableNT (node:internal/streams/readable:1698:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-05-10T18:23:01.678Z"}
{"code":"ERR_HTTP_HEADERS_SENT","level":"error","message":"Unhandled error: Cannot set headers after they are sent to the client","stack":"Error [ERR_HTTP_HEADERS_SENT]: Cannot set headers after they are sent to the client\n    at ServerResponse.setHeader (node:_http_outgoing:699:11)\n    at ServerResponse.header (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:794:10)\n    at ServerResponse.location (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:925:15)\n    at ServerResponse.redirect (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\response.js:963:18)\n    at file:///C:/Users/Punit/Desktop/ApiExplorer-Backend/src/routes/auth.routes.js:125:11\n    at Layer.handle [as handle_request] (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\express\\lib\\router\\route.js:149:13)\n    at complete (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\middleware\\authenticate.js:280:13)\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\middleware\\authenticate.js:287:15\n    at pass (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport\\lib\\authenticator.js:446:14)","timestamp":"2025-05-10T18:27:56.629Z"}
{"level":"error","message":"Unhandled error: Failed to obtain access token","stack":"Error: Failed to obtain access token\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\passport-oauth2\\lib\\strategy.js:178:49\n    at C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\oauth\\lib\\oauth2.js:214:7\n    at passBackControl (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\oauth\\lib\\oauth2.js:134:9)\n    at IncomingMessage.<anonymous> (C:\\Users\\Punit\\Desktop\\ApiExplorer-Backend\\node_modules\\oauth\\lib\\oauth2.js:157:7)\n    at IncomingMessage.emit (node:events:530:35)\n    at endReadableNT (node:internal/streams/readable:1698:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)","timestamp":"2025-05-10T18:28:05.932Z"}
